src/app/models/constraint-models/constraint.ts

```diff
@@ -10,6 +10,8 @@ import {ConstraintMark} from './constraint-mark';
 
 export class Constraint {
 
+  static readonly BASIC_DIMENSION: string = 'patient';
```

Comment:
maybe it is 'DEFAULT_DIMENSION'?

Reply:
shouldn't this be an enum?

Reply:
I named it `BASIC_DIMESENSION` to show that patient is on top of other dimensions in the hierarchy. It is indeed selected by default, so I am not against renaming it, if you think it suits better. Not sure about an enum, because we don't want to hardcode other dimensions, `patient` dimension is the only special one.

Reply:
why is patient different from the other dimensions?

Reply:
because GB needs to know that this is the top level dimension. It is the only one required (others are optional) and also it is used, when there is a need to fetch all the data (count calls, variable callse etc.) - wrapping in patient subselection

Reply:
just that naming it 'BASIC_DIMENSION' doesn't seem to imply that it is the top-level dimension, we could just name it 'TOP_LEVEL_DIMENSION'.